src/Symfony/Component/Security/Core/Tests/Validator/Constraints/UserPasswordValidatorTest.php

```diff
@@ -106,63 +111,53 @@ public function testUserIsNotValid()
 
     protected function createUser()
     {
-        $mock = $this->getMock('Symfony\Component\Security\Core\User\UserInterface');
+        $mock = $this->getMock(UserInterface::class);
 
         $mock
             ->expects($this->any())
             ->method('getPassword')
-            ->will($this->returnValue(static::PASSWORD))
-        ;
+            ->willReturn(static::PASSWORD);
 
         $mock
             ->expects($this->any())
             ->method('getSalt')
-            ->will($this->returnValue(static::SALT))
-        ;
+            ->willReturn(static::SALT);
 
         return $mock;
     }
 
     protected function createPasswordEncoder($isPasswordValid = true)
     {
-        return $this->getMock('Symfony\Component\Security\Core\Encoder\PasswordEncoderInterface');
+        return $this->getMock(PasswordEncoderInterface::class);
```

Comment:
this should be reverted. It makes merging branches harder (2.8 will keep using the string for the next 3 year of maintenance) and it does not add any real value to counterpart the maintenance pain